### PR TITLE
added packages/conf-lz4/conf-lz4.1.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**
also makes lz4_chans rely on it

### CODE REVIEW

File: packages/conf-lz4/conf-lz4.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+homepage: "http://lz4.org"
+authors: ["Yann Collet"]
+license: "GPLv2 & BSD-2-clause"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+depexts: [
```
Comment (kit-ty-kate) [inline packages/conf-lz4/conf-lz4.1.0.0/opam]: I would suggest to add a way of checking if the command is there. For example:
```suggestion
build: ["lz4" "-V"]
depexts: [
```

**Comment by kit-ty-kate** [comment]
could you also use it for previous versions of lz4_chans?

**Comment by UnixJunkie** [comment]
ok, updated.

**Comment by kit-ty-kate** [comment]
Thanks a lot!!

**Pull request merged**